packages/slider/src/useSlider.ts

```diff
@@ -225,6 +225,7 @@ export function useSlider<T extends Element = Element, M extends HTMLElement = H
             }
 
             if (value !== undefined) {
+              event.preventDefault(); // prevent screenreader navigation
```

Comment:
```suggestion
              event.preventDefault(); // avoid screenreader behavior that hinders keyboard navigation
```
nit: Mind tweaking the wording here a little? I don't think we want to give the impression that we're making this inaccessible to screen readers. 😅